src/util/CollisionDetection.js

```diff
@@ -0,0 +1,303 @@
+/*
+ * Paper.js - The Swiss Army Knife of Vector Graphics Scripting.
+ * http://paperjs.org/
+ *
+ * Copyright (c) 2011 - 2019, Juerg Lehni & Jonathan Puckey
+ * http://scratchdisk.com/ & https://puckey.studio/
+ *
+ * Distributed under the MIT license. See LICENSE file for details.
+ *
+ * All rights reserved.
+ */
+
+/**
+ * @name CollisionDetection
+ * @namespace
+ * @private
+ */
+var CollisionDetection = new (function() {
+    return /** @lends CollisionDetection */ {
```

Comment:
Since you don't actually have anything private inside the scope, you can also just use a normal object literal here:

```js
var CollisionDetection = {
    findItemBoundsCollisions: function() { …. }
};
```

Reply:
OK, will do.